certbot/bin/run_certbot.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
+# shellcheck disable=SC2039,SC2113,SC2039
+
+set -e
+
+print_usage() {
+cat << EOF
+usage: ${0} options
+
+This script helps to update content views in Foreman.
+
+OPTIONS:
+    Parameters:
+    -d Domain for certificate.
+    -e Administrator email used when obtaining certificates.
+    -r Webroot location to place challenge auth files.
+    -A Additional arguments to pass to certbot command.
+    -D Enable debug mode for additional output.
+    -S Enable staging mode for test certificates.
+
+Version: ${VERSION}
+EOF
+}
+
+function is_domain {
+    # Ensure cert_domain value is a hostname and not IP address.
```

Comment:
Comment doesn't make sense here.. maybe before line 34?

Reply:
I have comments at the start of all functions on what they should do. I think it makes sense to have the comment here.

Reply:
OK fair.. it was confusing because `cert_domain` isn't created until later and the text doesn't read like a function description.

Reply:
True, I've made the comment a bit more clear.